### ISSUE TITLE
Farore's Pearl appears in the wrong spot for some users

### ISSUE THREAD

**Issue by wooferzfg** [wooferzfg/tww-rando-tracker#1225]
![farore](https://github.com/wooferzfg/tww-rando-tracker/assets/8262173/be8b4ce4-24cf-4cf3-a4f6-2fc6f9333e8c)

**Comment by wooferzfg** [comment]
Tested this in Chrome, Edge, Safari, and Firefox, and it doesn't happen.

I was able to reproduce this issue by disabling `float: left` on the Din's Pearl element, but there's no reason for this to happen without some browser extension interfering with the CSS.
https://github.com/wooferzfg/tww-rando-tracker/assets/8262173/b3b8d282-a152-4d5d-a7cd-8551354381b8

Closing this issue until there's new information to indicate that this is actually a bug.
